### PR TITLE
Update node and clients docs page

### PR DESCRIPTION
**Pull request by ytrezq**
Don’t hesitate to fix grammar mistakes.

Add OpenEthereum trace api which is simpler to use than Ganache.
Add the fact that running your own node is a requirement for performing transaction frontrunning to Benefits.
Add Erion node (2 Tib disk space requirement for an archive node compared to 7.5Tib for Geth)

### CODE REVIEW

File: src/content/developers/docs/nodes-and-clients/index.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ If you want to [run your own node](/developers/docs/nodes-and-clients/run-a-node
 
 ### Archive node {#archive-node}
 
-- Stores everything kept in the full node and builds an archive of historical states. Needed if you want to query something like an account balance at block #4,000,000.
+- Stores everything kept in the full node and builds an archive of historical states. Needed if you want to query something like an account balance at block #4,000,000 or simply and realiablely [test your own transactions set without mining them using OpenEthereum](https://openethereum.github.io/JSONRPC-trace-module#trace_callmany).
```
Comment (wackerow) [inline src/content/developers/docs/nodes-and-clients/index.md:45]: - Would add comms after `#4,000,000`
- "realiablely" -> "reliably"

File: src/content/developers/docs/nodes-and-clients/index.md [hunk 2]
```diff
@@ -55,6 +55,8 @@ Running a node allows you to trustlessly and privately use Ethereum while suppor
 
 Running your own node enables you to use Ethereum in a truly private, self-sufficient and trustless manner. You don't need to trust the network because you can verify the data yourself with your client. "Don't trust, verify" is a popular blockchain mantra.
 
+- You can run the programs using your node on the same computer. This allows you to connect to your node using IPC or possibly through rewriting the node to load your program as a plugin bringing low latency which is a requirement when you to replace your transactions as fast as possible (aka frontrunning).
```
Comment (wackerow) [inline src/content/developers/docs/nodes-and-clients/index.md:58]: Not sure I fully understand this. Is it a point about being able to investigate the blockchain locally on your computer? Or about frontrunning? If both, I would consider breaking into two, and placing the frontrunning information bottom on the list as it's a more advanced topic.

File: src/content/developers/docs/nodes-and-clients/index.md [hunk 3]
```diff
@@ -55,6 +55,8 @@ Running a node allows you to trustlessly and privately use Ethereum while suppor
...
+- You can program your own custom RPC endpoints.
 - Your node verifies all the transactions and blocks against consensus rules by itself. This means you don’t have to rely on any other nodes in the network or fully trust them.
```
Comment (wackerow) [inline src/content/developers/docs/nodes-and-clients/index.md:60]: I would consider leaving this line first in the list (`Your node verified all the...`)

**Review by wackerow — CHANGES_REQUESTED** [review]
@ytrezq Thanks for dropping this 👍🏻 Just a couple comments to get it over the line =)

File: src/content/developers/docs/nodes-and-clients/index.md (hunk 2 above)
Comment (ytrezq) [reply src/content/developers/docs/nodes-and-clients/index.md:58]: @wackerow this is only about frontrunning. I’m meaning, the ability to detect and replace transactions as fast as possible.

File: src/content/developers/docs/nodes-and-clients/index.md (hunk 3 above)
Comment (ytrezq) [reply src/content/developers/docs/nodes-and-clients/index.md:60]: @wackerow I approved you for doing the commit.

File: src/content/developers/docs/nodes-and-clients/index.md (hunk 1 above)
Comment (ytrezq) [reply src/content/developers/docs/nodes-and-clients/index.md:45]: I approved you for doing the commit.

**Comment by minimalsm** [comment]
Hey @ytrezq! 

Anything I can do to help get this over the line 🏃‍♂️?

**Comment by minimalsm** [comment]
How can I get a new deploy preview from Netlify?

**Review by wackerow — APPROVED** [review]
👍🏻

**Comment by minimalsm** [comment]
Awesome thanks for the PR @ytrezq ✌️.

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @ytrezq for docs

**Pull request merged**